environment.devenv.yml

```diff
@@ -1,6 +1,7 @@
 name: autodiff
 
 dependencies:
+  - black
```

Comment:
What does this dependency do?

Reply:
This a formatter for Python codes. I thought that I removed it, this is just a leftover. Thanks!